src/browser/api/application.js

```diff
@@ -438,14 +438,14 @@ Application.revokeWindowAccess = function() {
     console.warn('Deprecated');
 };
 
-Application.run = function(identity, configUrl = '') {
+Application.run = function(identity, userAppConfigArgs, configUrl = '') {
```

Comment:
Generally speaking, new params must be added to end of existing param list to maintain backwards compatibility